scripts/duplicate-wavs-altraces-variants.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Check if bnk with matching prefix exists and creates a renamed duplicat for variants
+# Vars: 1. Race prefix, 2. Gender prefix, 3. Event, 4.File
+create_alt_copies() {
+	event="${3%.wav}"
+	
+	# Check for normal, altvoice and beggar variant
+	for prefix in "" "_altvoice" "_beggar"
```

Comment:
Leider sehr langsam. Idealerweise, würden wir das auch so ähnlich wie in einem der anderen Skripts machen - also mit einer `find`, `sed`, `xargs` pipe:
```bash
find sound2wem/Windows/ -name "*.wem" | sed "s/\.wem$//" | xargs -P 64 -I {} ./voxmeld/voxmeld.exe {}
```